src/gettingstarted/own-code/create-project.md

```diff
@@ -0,0 +1,57 @@
+# Import your own code
+
+## Create a new project
+
+With the Platform.sh CLI now installed and configured to communicate with your projects, you can create a new project from the command line and connect it to your application.
+
+<asciinema-player src="/videos/asciinema/project-create.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Create an empty project**
+
+    Type the command `platform create` in your terminal.
+
+    The CLI will then ask you to set up some initial project configurations:
+
+    * `Project title`: We need a unique name for each project, so title this one `My CLI Project`.
```

Comment:
"You", not "We".

Also, technically duplicate human-friendly titles are allowed, they're just horribly confusing.